src/libraries/System.Text.RegularExpressions/tests/FunctionalTests/Regex.Match.Tests.cs

```diff
@@ -149,6 +149,15 @@ private static IEnumerable<(string Pattern, string Input, RegexOptions Options,
                     yield return (Case("(?>[^z]+)z"), "zzzzxyxyxyz123", options, 4, 9, true, "xyxyxyz");
                     yield return (Case("(?>(?>[^z]+))z"), "zzzzxyxyxyz123", options, 4, 9, true, "xyxyxyz");
                     yield return (Case("(?>[^z]*)z123"), "zzzzxyxyxyz123", options, 4, 10, true, "xyxyxyz123");
+                    yield return (Case("(?>a*)a"), "aaa", options, 0, 3, false, "");
+                    yield return (Case("(?>a*)a+"), "aaa", options, 0, 3, false, "");
```

Comment:
```suggestion
                    yield return (Case("(?>a*)a+"), "aaa", options, 0, 3, false, "");
                    yield return (Case("(?>a*)a{1,}"), "aaa", options, 0, 3, false, "");
```
etc ... or are these equivalent?

Reply:
I guess they are and that's protected by PatternsReduceIdentically tests.

Reply:
Correct